lib/src/local_working_copy.rs

```diff
@@ -676,6 +706,7 @@ fn file_state(metadata: &Metadata) -> Option<FileState> {
             file_type,
             mtime,
             size,
+            materialized_conflict_data,
```

Comment:
nit: maybe better to copy conflict data only if the new file type is a normal file? 

(but see also the other comment. this could be handled by `process_present_file()`.)